dm/master/scheduler/scheduler_test.go

```diff
@@ -410,6 +420,11 @@ func (t *testScheduler) testSchedulerProgress(c *C, restart int) {
 	rebuildScheduler(ctx)
 
 	// CASE 4.6: try remove source when subtasks exist.
+	// wait for source bound recovered
+	c.Assert(utils.WaitSomething(30, 100*time.Millisecond, func() bool {
+		bounds := s.BoundSources()
+		return len(bounds) == 2 && bounds[0] == sourceID1 && bounds[1] == sourceID2
```

Comment:
will it has another bound order? @csuzhangxc

Reply:
no, we sorted in `BoundSources`.